docs/wiki/introduction/sql.md

```diff
@@ -305,12 +305,12 @@ The following trig functions: `sin`, `cos`, `tan`, `cot`, `asin`, `acos`, `atan`
 
       osquery> .mode line
 
-      osquery> select uid from users;
+      osquery> select uid from users;  
```

Comment:
Is this whitespace intentional?

Reply:
Yes, it's to force a linebreak, since if you check here https://osquery.readthedocs.io/en/latest/introduction/sql/#string-functions, the `uid = 500` line is rendered on the same line of the query.